src/helper/types/transcription_data.py

```diff
@@ -68,3 +77,17 @@ class TranscriptionPostResponse(BaseModel):
     language: str
 
 
+class TranslationPostData(TypedDict):
```

Comment:
Is there a reason for this to be a typed dict instead of a basemodel? Does this work properly with swagger nonetheless?